docs/architecture/mobile-clients/ios/push-notification-troubleshooting.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+sidebar_position: 2
+---
+
+# iOS Push Notification Troubleshooting Tips
+
+## Overview
+
+We use push notifications in the Password Manager app for two purposes:
+
+1. To sync the vault when a change was made from a different devices
+2. To allow users to log in with device
+
+However, there are a lot of things that can go wrong with push notifications, and the process is
+relatively opaque. This document will provide some places where you can dig in to things under the
+hood and hopefully figure out what’s going on.
+
+## Triggering a Push Notification
+
+The easiest way to trigger a push notification is to be logged in on both the app and a web vault,
+and to make a change in the web vault. The server should send a push notification to the device, and
```

Comment:
🎨 May be misinterpreted to suggest that other changes like settings trigger push notifications too. Given that the Overview section already lists how push is used, this part could provide a specific action we usually use.  
 
```suggestion
and to edit a cipher in the web vault. The server should send a push notification to the device, and
```

Reply:
Ah, makes sense, though I'll probably do it as add, edit, or delete a cipher (since they all should trigger the push) 🤔